src/nvim/api/vim.c

```diff
@@ -2250,6 +2250,56 @@ Array nvim_get_proc_children(Integer pid, Error *err)
   return rvobj;
 }
 
+Boolean nvim_add_autocommand(
+    uint64_t channel_id,
+    String event, String pat, DictionaryOf(LuaRef) opts, Error *err
```

Comment:
Here we'd like to use just plain callback as the last argument. We are missing API harness for `LuaRef` though right now.